spyder/utils/ipython/spyder_kernel.py

```diff
@@ -138,9 +138,9 @@ def get_var_properties(self):
                     'array_ndim': self._get_array_ndim(value)
                 }
 
-            return properties
+            return repr(properties)
         else:
-            return {}
+            return repr({})
```

Comment:
Sorry, this should be `repr(None)` to make the objects emitted when debugging the same as when users are not debugging. The last thing is done in `handle_exec_method`, here:

https://github.com/spyder-ide/spyder/blob/3.x/spyder/widgets/ipythonconsole/shell.py#L338